### DISCUSSION TITLE
Does the codebase support Sleepy End Devices?

### DISCUSSION THREAD

**Discussion by darianbjohnson** [SiliconLabs/arduino#90] [General]
I saw a note is the issue log stating that support for Sleepy End Devices was on the roadmap. Wasn't sure if it was implemented yet.

**Comment by ggtimtom** [comment]
I think it come with the 1.4 release: [https://csa-iot.org/newsroom/matter-1-4-enables-more-capable-smart-homes/](https://csa-iot.org/newsroom/matter-1-4-enables-more-capable-smart-homes/)

**Comment by ggtimtom** [comment]
Enhancements to Battery Powered Devices:
Matter 1.4 introduces core enhancements that optimize battery life and communication for Intermittently Connected Devices (ICDs) like switches, buttons, and sensors. Key updates include the Long Idle Time (LIT) protocol, which extends battery life, and a new Check-In Protocol to ensure reliable communication for low-power devices that require LIT. Additionally, quieter reporting of predictable attribute changes reduces network traffic, resulting in improved battery performance and network efficiency.

Developers interested in learning more about these enhancements can access the following resources:

[Matter 1.4 specification and test plan](https://csa-iot.org/developer-resource/specifications-download-request/)
[Matter 1.4 SDK Release](https://github.com/project-chip/connectedhomeip)

**Comment by darianbjohnson** [comment]
Hi - surfacing this post. Is there a plan/timeline for Intermittedly Connected Devices (formerly Sleepy Devices) to be supported in the Arduino codebase?

**Comment by lboue** [comment]
I am also interested in this feature.
